index.html

```diff
@@ -7,32 +8,34 @@
 </head>
 <body>
 
-<pre id="paste">
-                         _
-     ___ ___ _____ _____|_|___  <span class="sickle">☭</span>
-    |  _| . |     |     | | -_|
-    |___|___|_|_|_|_|_|_|_|___|
-
-    commie is a pastebin script with line commenting support.
-    available at <a href="https://github.com/splitbrain/commie">github</a>. made by <a href="http://www.splitbrain.org">splitbrain</a>.
-
+<div id="content">
 
+<pre>
+  _____                 _
+ |     |___ _____ _____|_|___  <span class="sickle">☭</span>
+ |   --| . |     |     | | -_|
+ |_____|___|_|_|_|_|_|_|_|___|
+</pre>
 
+<div id="paste">
+commie is a pastebin script with line commenting support.<br>
+available at <a href="https://github.com/splitbrain/commie">github</a>. made by <a href="http://www.splitbrain.org">splitbrain</a>.
+</div>
 
-</pre>
 <div id="help">☛ Click the line numbers to comment</div>
 
 <div id="new">
     <label for="newpaste">Create a new paste:</label>
     <textarea id="newpaste"></textarea><br/>
     <button>Save new Paste</button>
-    Remember: everything is public!
+    <span class="warn">Remember: everything is public!</span>
 </div>
 
 <div id="footer">
-    powered by <a href="">commie<sup>☭</sup></a>
+    powered by <a href="http://commie.io">commie<sup>☭</sup></a> <br> 2015-<script>document.write(new Date().getFullYear())</script>
```

Comment:
document.write() is bad style. This should be properly added through DOM modification.